conversion/conversion.go

```diff
@@ -1060,10 +1061,11 @@ func SetRowStats(driver string, conv *internal.Conv, db *sql.DB) error {
 // ProcessSQLData invokes ProcessSQLData function from a sql package based on driver selected.
 func ProcessSQLData(driver string, conv *internal.Conv, db *sql.DB) error {
 	switch driver {
+	//TODO - move this logic into a factory within the sources dir
```

Comment:
I think this TODO makes sense, but wasn't sure whether this was stale since I don't see a similar TODO in other methods above.

Reply:
Marked it in one place as a placeholder and reminder. Needs to be done at all invocation points